pkg/aws/client.go

```diff
@@ -970,6 +986,28 @@ func (c *awsClient) CreateS3Bucket(bucketName string, region string) error {
 	if err != nil {
 		return err
 	}
+
+	_, err = c.s3Client.PutPublicAccessBlock(&s3.PutPublicAccessBlockInput{
+		Bucket: aws.String(bucketName),
+		PublicAccessBlockConfiguration: &s3.PublicAccessBlockConfiguration{
+			BlockPublicAcls:       aws.Bool(true),
+			IgnorePublicAcls:      aws.Bool(true),
+			BlockPublicPolicy:     aws.Bool(false),
+			RestrictPublicBuckets: aws.Bool(false),
+		},
+	})
+	if err != nil {
+		return err
+	}
+
+	_, err = c.s3Client.PutBucketPolicy(&s3.PutBucketPolicyInput{
```

Comment:
ROSA runs within client side, so in general it may have all permissions as long as the company is not restricting access. If this happens that's why we have manual mode where they give the commands with someone who has permission

Reply:
That is what I was thinking, do you know if we have it documented anywhere the permissions needed to actually fun the CLI?